flint.ui/src/store/modules/point.js

```diff
@@ -221,19 +201,22 @@ export default {
           this._vm.$toast.success(`${response}`, { timeout: 2000 })
           console.log(response)
           //this.state.received_data = response.data;
-          commit('set_received_point_example_data', response.data)
-          console.log(this.state.received_data)
+          commit('save_results', response.data)
+          console.log(this.state.point.results)
         })
         .catch((error) => {
           this._vm.$toast.error(`${error}`, { timeout: 2000 })
           console.log(error)
         })
     },
     process_point_config({ commit }) {
```

Comment:
Can you please explain when `{ commit }` is used?

Reply:
`{ commit }` is used to commit a mutation. Here the `'save_results'` mutation stores the `response.data` into the store. This code runs when we click on the Run button in the Stepper.